docs/authentication/saml/authentication-flows.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Authentication flows
+description: Learn about the SAML SSO Authentication flows.
+---
+
+# Authentication flows
+
+The SAML protocol supports two different methods in order to start an SSO flow: SP-initiated and IdP-initiated. Below we will explore
+how each of them works and the differences between the two.
+
+## SP-initiated flow
+In an SP-initiated flow, the user is starting the authentication flow from your application (SP), by providing the email address. Then they are being redirected to the SAML provider (IdP) where
+they need to authenticate themselves. After successful authentication, the user is finally redirected back to your application, gaining access to their account.
```

Comment:
```suggestion
- After successful authentication, the user is redirected back to your application, gaining access to their account.
```

Reply:
We should mention what happens when authentication fails, for completeness

Reply:
Nothing fancy and it depends on the SAML provider. I guess the most common scenario is that the provider just returns an error on their sign in page. I guess it's fine to focus only on the happy path here?

Reply:
Ah, I see. That's fine then yeah